apis/python/src/tiledb/vector_search/flat_index.py

```diff
@@ -40,6 +40,7 @@ def __init__(
             + self.index_version
         ].uri
         schema = tiledb.ArraySchema.load(self.db_uri, ctx=tiledb.Ctx(self.config))
+        self.expected_query_columns = schema.shape[0]
```

Comment:
nit: change the name to dimensions

Reply:
Sure, done.

Reply:
Thanks, please use just `self.dimensions` this variable might be important for testing in other parts of the code also.